src/main/webapp/script.js

```diff
@@ -123,24 +123,25 @@ function jsListToHtml(list){
 
     for(element of list)
     {
-      const checked = element.status=="closed"?'checked':"";
+      const checked = element.status==="closed"?'checked':"";
       html += `<input type="checkbox" name="${element.id}" value="${element.message}" ${checked} 
                 onclick="updateRecordStatus(event.target)"/>
-      <label for="${element.id}">${element.message}</label><br><br>`;
+      <label for="${element.id}">${element.message}</label>
```

Comment:
A little confused by the syntax here, could you explain it in a little mor detail?